gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -129,12 +146,13 @@ def __init__(self, args):
     self.temp_location = args.temp_location
     self.logging_location = args.logging_location
     self.project = args.project
-
-  def __enter__(self):
     self.credentials = GoogleCredentials.get_application_default()
-    client = bigquery.Client(project=self.project)
     self.dataset_id = 'integration_tests_{}'.format(
         datetime.now().strftime('%Y%m%d_%H%M%S'))
+    self.dataset = None
+
+  def __enter__(self):
+    client = bigquery.Client(project=self.project)
     dataset_ref = client.dataset(self.dataset_id)
     # TODO(bashir2): There is a mismatch between the bigquery API versions
```

Comment:
I think you can remove this TODO now. BTW, I thought we need a version strictly greater than 0.25.

Reply:
Done. And good catch! I changed to be >0.25.